src/Resources/config/intl.php

```diff
@@ -61,79 +63,71 @@
 
         ->set('sonata.intl.locale_detector.request_stack', RequestStackDetector::class)
             ->public()
+            ->deprecate(...BCDeprecationParameters::forConfig(
+                'The "%service_id%" service is deprecated since sonata-project/intl-bundle 2.13 and will be removed in 3.0.',
+                '2.13'
+            ))
             ->args([
                 new ReferenceConfigurator('request_stack'),
                 '',
             ])
 
-        ->set('sonata.intl.templating.helper.locale', '%sonata.intl.templating.helper.locale.class%')
+        ->set('sonata.intl.helper.locale', '%sonata.intl.helper.locale.class%')
             ->public()
-            ->tag('templating.helper', [
-                'alias' => 'locale',
-            ])
             ->args([
                 '%kernel.charset%',
-                new ReferenceConfigurator('sonata.intl.locale_detector'),
             ])
 
-        ->set('sonata.intl.templating.helper.number', '%sonata.intl.templating.helper.number.class%')
+        ->set('sonata.intl.helper.number', '%sonata.intl.helper.number.class%')
             ->public()
-            ->tag('templating.helper', [
-                'alias' => 'number',
-            ])
             ->args([
                 '%kernel.charset%',
-                new ReferenceConfigurator('sonata.intl.locale_detector'),
             ])
 
-        ->set('sonata.intl.templating.helper.datetime', '%sonata.intl.templating.helper.datetime.class%')
```

Comment:
Removing a service alias is a BC break (or this kind of change *was handled* as a BC break a few years ago...)